cli/preprocessor.go

```diff
@@ -100,9 +98,9 @@ func analyze(
 	} else {
 		// Ensure value of comment char
 		if *lineCommentIden == "" && *blockCommentIdenOpen == "" && *blockCommentIdenClose == "" {
-			log.Fatal("You must provide at least one of line comment or block comments identifiers.")
+			util.Error("You must provide at least one of line comment or block comments identifiers.", true)
```

Comment:
what's the `true` argument for?

Reply:
Look here:
https://github.com/compose-generator/ccom/pull/44/files#diff-15a25de69b863b267217c2db6ca7b09a36fe3dc0c5f36a02b031bf16c5681124R32-R34